src/lang/Messages.properties

```diff
@@ -1876,12 +1876,20 @@ sites.spider.popup         = Spider...
 sites.showinsites.popup    = Show in Sites Tab
 
 spider.activeActionPrefix = Spidering: {0}
+spider.api.action.addDomainAlwaysInScope = Adds a new domain that's always in scope.
+spider.api.action.modifyDomainAlwaysInScope = Modifies a domain that's always in scope.
```

Comment:
What does this actually mean? Might be worth explaining a bit more + giving an overview of the params?